app/utils/confirmRegistration/confirmRegistration.factory.js

```diff
@@ -0,0 +1,58 @@
+(() => {
+
+    'use strict';
+
+    angular
+        .module('app')
+        .factory('ConfirmRegistrationFactory', ConfirmRegistrationFactory);
+
+    ConfirmRegistrationFactory.$inject = ['$resource'];
+
```

Comment:
Please, remove empty line. Will be more comfortable for seeing if you keep function description and function injection together.